1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -20,21 +20,41 @@ instead!
    Use `console.log` to display the result.
 -----------------------------------------------------------------------------*/
 const cartForParty = {
-  // TODO complete this object
+  // TODO complete this function
+  beers: 1.75,
+  chips: 0.99,
+  chocolate: 1.25,
+  waffles: 2.15,
+  tea: 1.5,
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
+function calculateTotalPrice(cartForParty) {
   // TODO replace this comment with your code
+  let amount = 0;
+  let x;
+  for (x in cartForParty) {
+    amount += cartForParty[x];
+  }
+  return console.log('"Total: €' + amount.toFixed(2) + '"');
```

Comment:
Here you made a mix of console.log and returning. So close! Try this:

```suggestion
  return 'Total: €' + amount.toFixed(2);
```

Or, if you really want to console log too, you can do both:

```suggestion
  console.log('Total: €' + amount.toFixed(2));
  return 'Total: €' + amount.toFixed(2);
```